diff --git a/src/bfio/utils.py b/src/bfio/utils.py
index 954d682..cb88332 100644
--- a/src/bfio/utils.py
+++ b/src/bfio/utils.py
@@ -40,6 +40,8 @@ try:
 
         global JAR_VERSION
         scyjava.config.endpoints.append("ome:formats-gpl:8.0.0")
+        scyjava.config.set_m2_repo("/etc/m2/")
+        scyjava.config.set_cache_dir("/etc/m2/jgo_cache")
         scyjava.start_jvm()
         import loci
 
